### DISCUSSION TITLE
RTDB Upload Question

### DISCUSSION THREAD

**Discussion by parkkyeo** [GogoVega/node-red-contrib-firebase-realtime-database#78] [Q&A]
Does repeatedly uploading the same value to Firebase RTDB affect the data upload usage?

I want to upload data to RTDB only when the value changes. Is there a way to do this?

**Comment by GogoVega** [comment]
Hi,
The content of your DB is downloaded locally.

 If you set the same value multiple times, only the first change will be synchronized with the remote DB because after that there are no more changes - it's the same value.

If you use a `Firebase IN` node you can listen for changes.
